srv/components/misc_pkgs/ipmi/bmc_watchdog/install.sls

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+Install bmc-watchdog:
+  pkg.installed:
+    - name: freeipmi-bmc-watchdog
+
+Update watchdog timeout:
+  file.replace:
+    - name: /etc/sysconfig/bmc-watchdog
+    - pattern: '900'
+    - repl: '300'
+
+Reload kernel with Modprobe:
+  file.managed:
+    - name: /etc/modules
```

Comment:
I don't have a good feeling about this file: `/etc/modules`
Change this to: `/etc/modules-load.d/ipmi_mods.conf`
Ref: https://unix.stackexchange.com/a/189696

Reply:
This is an optional step to reload the kernel before ipmi command can be run. And it accepts the options on /modules file and doesnt not check in any .conf files. So, I created one new and added. I'll test this on HW anyway and see if it works on .conf.